src/connectivity.jl

```diff
@@ -536,17 +536,27 @@ end
 """
     isgraphical(degs)
 
-Return true if the degree sequence `degs` is graphical, according to
-[Erdös-Gallai condition](http://mathworld.wolfram.com/GraphicSequence.html).
+Return true if the degree sequence `degs` is graphical
```

Comment:
There should be in the sentence such as "A sequence of integers is called *graphical*, if there exists a graph where the degrees of its vertices form that same sequence" or something easier if you have a good idea for that.

I would suggests that the description of the Erdös-Gallai-theorem should be moved to a `Implementation` subsection, as this is not important for someone who just wants to know what the function outputs.